kai/rpc_server/server.py

```diff
@@ -622,18 +622,20 @@ def get_codeplan_agent_solution(
         app.log.debug("QUEUE_STATE_END_OF_CODE_PLAN: IGNORED_TASKS: END")
 
         msg = f"Finished with {len(overall_result.encountered_errors)} errors and {len(overall_result.modified_files)} modified files.\n"
-        msg += "<details><summary>Successful tasks:</summary>\n"
-        msg += "<ul>\n"
-        for task in app.task_manager.processed_tasks - initial_solved_tasks:
-            msg += f"<li>{task}</li>\n"
-        msg += "</ul>\n"
-        msg += "</details>\n"
-        msg += "<details><summary>Ignored tasks:</summary>\n"
-        msg += "<ul>\n"
-        for task in set(app.task_manager.ignored_tasks) - initial_ignored_tasks:
-            msg += f"<li>{task}</li>\n"
-        msg += "</ul>\n"
-        msg += "</details>\n"
+        if app.task_manager.processed_tasks - initial_solved_tasks:
```

Comment:
can we make this a function somehow using the markdown version of the tasks?

Reply:
@shawn-hurley this is a long list. not sure whether we want to use markdown in there

Reply:
I guess I was hopping you could make it a drop down

I am good with a follow up